{{cookiecutter.project_name}}/tox.ini

```diff
@@ -12,17 +12,16 @@ skipsdist = True
 envlist =
     # always keep coverage-clean first
     coverage-clean
-    lint
-    flake8
-    mypy
+    lint-fix
+    codespell-write
```

Comment:
It it worth having `lint` as part of the default envlist as well? `lint-fix` might not fix everything so maybe it's worth letting the developer know that there is more stuff for them to fix manually?

Reply:
Makes sense ... adding.